src/apps/GlobalApp/GlobalApp.tsx

```diff
@@ -1,19 +1,42 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import { Header, Footer, NavLink, PageSection, Banner, About, History, CtaButtons, Loading } from "../../components";
+import { Header, Footer, NavLink, PageSection, Banner, About, History, CtaButtons, Loading, CountDown } from "../../components";
 import { FlagArgentina, FlagChile, FlagMexico, FlagUruguay } from "../../components/SVGs";
 import { BlogPage, ConductPage, SpeakersPage, SponsorsPage, ExecutiveTeamPage } from "../../pages";
 import { IEdition } from "../../types/IEdition";
-import { resourcesService, backendService } from "../../services";
+import { siteService , resourcesService, backendService } from "../../services";
 import Constants from "../../constants";
 
 import styles from "./GlobalApp.module.scss";
 
 const Home: React.SFC<any> = ({ conferenceInfo }: { conferenceInfo: IEdition }) => {
   const Resources = resourcesService.getResources();
 
+  const getTranslation = (key: string): string => {
+    switch (key) {
+      case 'days':
+        return Resources.info.days;
+      case 'hours':
+        return Resources.info.hours;
+      case 'minutes':
+        return Resources.info.minutes;
+      case 'seconds':
+        return Resources.info.seconds;
+      case 'areYouReady':
+        return Resources.info.areYouReady;
+      case 'vOpenGlobalConference':
+        return Resources.titles.vOpenGlobalConference;
+      default:
+        return key;
+    }
+  }
+
   return (
     <>
+      {
+        siteService.mustSetUpCountDown() &&
+        <CountDown getTranslation={getTranslation} deadline={new Date(2020, 9, 3, 9, 0, 0)} current={new Date()}/>
```

Comment:
Why don't you move the `getTranslation` inside of the Countdown component? It looks like it's logic for this component only.